proto/dclauth/revoked_account.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package zigbeealliance.distributedcomplianceledger.dclauth;
+
+option go_package = "github.com/zigbee-alliance/distributed-compliance-ledger/x/dclauth/types";
+
+import "gogoproto/gogo.proto";
+import "dclauth/account.proto";
+import "dclauth/grant.proto";
+
```

Comment:
We need to extend the documentation: please add new commands to query an account to https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/docs/transactions.md#auth.

Reply:
Done! ✅